core/src/player.rs

```diff
@@ -480,46 +480,47 @@ impl Player {
             // Sanity: If we had too many frames to tick, just reset the accumulator
             // to prevent running at turbo speed.
             if self.frame_accumulator >= frame_time {
-                self.frame_accumulator = 0.0;
+                self.frame_accumulator = Duration::ZERO;
             }
 
             // Adjust playback speed for next frame to stay in sync with timeline audio tracks ("stream" sounds).
             let cur_frame_offset = self.frame_accumulator;
-            self.frame_accumulator += self.mutate_with_update_context(|context| {
+
+            let add = self.mutate_with_update_context(|context| {
```

Comment:
nit: Can this be restored?

Reply:
It produces the following compile error.

I see this kind of problem rarely, and I don't know of any solution other than creating a temporary variable.
Do you know of a better way?

```rust
error[E0499]: cannot borrow `*self` as mutable more than once at a time
   --> core/src/player.rs:489:39
    |
489 |                self.frame_accumulator += self.mutate_with_update_context(|context| {
    |   _____________----------------------____^
    |  |             |
    |  |_____________first mutable borrow occurs here
    | ||
490 | ||                 context
491 | ||                     .audio_manager
492 | ||                     .audio_skew_time(context.audio, cur_frame_offset)
493 | ||             });
    | ||              ^
    | ||______________|
    | |_______________second mutable borrow occurs here
    |                 first borrow later used here
    |
help: try adding a local storing this...
   --> core/src/player.rs:489:39
    |
489 |               self.frame_accumulator += self.mutate_with_update_context(|context| {
    |  _______________________________________^
490 | |                 context
491 | |                     .audio_manager
492 | |                     .audio_skew_time(context.audio, cur_frame_offset)
493 | |             });
    | |______________^
help: ...and then using that local here
   --> core/src/player.rs:489:13
    |
489 | /             self.frame_accumulator += self.mutate_with_update_context(|context| {
490 | |                 context
491 | |                     .audio_manager
492 | |                     .audio_skew_time(context.audio, cur_frame_offset)
493 | |             });
    | |______________^

```